config.yml

```diff
@@ -26,12 +26,29 @@ genesis:
         denom: uelys
     gov:
       deposit_params:
-        max_deposit_period: 20s
+        max_deposit_period: 172800s
```

Comment:
Is there any specific reason increasing this max deposit period?

Reply:
Ah I have forgotten to reduce that. Actually it was 10s from Caner. We should increase it anyways.

Reply:
@kenta-elys as discussed lets keep it back at 20s for now thanks

Reply:
@cosmic-vagabond Done.